app/controllers/comments_controller.rb

```diff
@@ -14,6 +19,10 @@ def create
   end
 
   def destroy
+    @notification = Notification.where(:comment_id == @comment.id)
```

Comment:
oh; I think I get it; you want to destroy all of the notifications on a comment before destroying the comment itself.

that should go into your comment model actually; see [dependent associations](http://api.rubyonrails.org/classes/ActiveRecord/Associations/ClassMethods.html#module-ActiveRecord::Associations::ClassMethods-label-Dependent+associations)